src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Infrastructure/TaskExtensions.cs

```diff
@@ -151,7 +99,7 @@ public static async Task<T> WithTimeout<T>(this Task<T> instance, TimeSpan timeo
             // signal the caller.
 
             cancellationToken?.Cancel();
-            instance.FireAndForget();            
+            var _ = instance.ConfigureAwait(false);
```

Comment:
Why is this required?

Reply:
yeah... fair point.  That's already running.  My bad.